packages/openapi-fetch/src/index.test.ts

```diff
@@ -48,7 +48,7 @@ describe("client", () => {
 
       // data
       mockFetchOnce({ status: 200, body: JSON.stringify(["one", "two", "three"]) });
-      const dataRes = await client.GET("/string-array", {});
+      const dataRes = await client.GET("/string-array");
```

Comment:
Note: the removal of all `, {})` in the test suite is a good sign. The test suite has quite a few `// @ts-expect-error` comments which are vital to testing the type assertions.

But in addition, the test suite also has runtime tests to ensure the function is behaving as-expected. The fact that no tests were modified, while still passing, is a great sign.